### PR TITLE
publish to maven central

### PR DESCRIPTION
**Pull request by nbransby**

### CODE REVIEW

File: firebase-auth/package.json [hunk 1]
```diff
@@ -8,20 +8,20 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/TeamHubApp/firebase-kotlin-multiplatform-sdk.git"
+    "url": "git+https://github.com/GitLiveApp/firebase-kotlin-multiplatform-sdk.git"
```
Comment (nbransby) [inline firebase-auth/package.json:11]: wrong url

File: build.gradle.kts [hunk 2]
```diff
@@ -1,29 +1,150 @@
...
+        into("$rootDir/firebase-database/src/iosMain/c_interop/modules/FirebaseFirestore.framework")
+    }
+
+    val copyFirebaseFunctions by creating(Copy::class){
+        dependsOn(copyFirebaseFirestore)
+        from("$rootDir/Firebase/FirebaseFunctions/FirebaseFunctions.framework")
+        into("$rootDir/firebase-database/src/iosMain/c_interop/modules/FirebaseFunctions.framework")
+    }
+
+    val copyAllFirebaseFrameworks by creating(Copy::class){
```
Comment (nbransby) [inline build.gradle.kts:62]: give these ios names

File: build.gradle.kts [hunk 3]
```diff
@@ -1,29 +1,150 @@
...
+    }
+
+    val copyFirebaseAuth by creating(Copy::class){
+        dependsOn(unzipFirebase)
+        from("$rootDir/Firebase/FirebaseAuth/FirebaseAuth.framework")
+        into("$rootDir/firebase-auth/src/iosMain/c_interop/modules/FirebaseAuth.framework")
+    }
+
+    val copyFirebaseDatabase by creating(Copy::class){
+        dependsOn(copyFirebaseAuth)
```
Comment (nbransby) [inline build.gradle.kts:45]: does make much sense depending on copyFirebaseAuth

File: build.gradle.kts [hunk 4]
```diff
@@ -1,29 +1,150 @@
...
+            from(file("$buildDir/classes/kotlin/js/main/${project.name}.js.map"))
+            into(file("$buildDir/node_module"))
+        }
+
+
+        val publishToNpm by creating(Exec::class) {
+            
+            if(!File(rootDir, "Firebase").exists()) {
+                dependsOn(
+                    rootProject.tasks.named("copyAllFirebaseFrameworks").get(),
```
Comment (nbransby) [inline build.gradle.kts:129]: seperate these out as they are related one is ios while other is js, and the task is publishToNpm

File: firebase-database/build.gradle.kts [hunk 5]
```diff
@@ -75,7 +60,7 @@ kotlin {
                 val firebaseDatabase by cinterops.creating {
                     packageName("cocoapods.FirebaseDatabase")
                     defFile(file("$projectDir/src/iosMain/c_interop/FirebaseDatabase.def"))
-                    compilerOpts("-F$projectDir/src/iosMain/c_interop/modules/FirebaseDatabase-6.17.0")
```
[the remark this answers is not in the document]
Comment (sadevelopment) [reply firebase-database/build.gradle.kts:78]: Think this is an old checkin, its not that anymore

File: firebase-auth/package.json (hunk 1 above)
Comment (sadevelopment) [reply firebase-auth/package.json:11]: This isn't there anymore it in the root gradle->subprojects

Comment (sadevelopment) [reply firebase-auth/package.json:11]: Done in all package.json's

File: build.gradle.kts (hunk 2 above)
Comment (sadevelopment) [reply build.gradle.kts:62]: Done

**Pull request merged**